src/Integrator/Flame.cpp

```diff
@@ -32,29 +32,29 @@ Flame::Parse(Flame& value, IO::ParmParse& pp)
         pp_query("timestep", value.base_time);
         // These are the phase field method parameters
         // that you use to inform the phase field method.
-        pp_query("pf.eps", value.pf.eps); // Burn width thickness
-        pp_query("pf.kappa", value.pf.kappa); // Interface energy param
-        pp_query("pf.gamma", value.pf.gamma); // Scaling factor for mobility
-        pp_query("pf.lambda", value.pf.lambda); // Chemical potential multiplier
-        pp_query("pf.w1", value.pf.w1); // Unburned rest energy
-        pp_query("pf.w12", value.pf.w12);  // Barrier energy
-        pp_query("pf.w0", value.pf.w0);    // Burned rest energy
-        pp_query("amr.ghost_cells", value.ghost_count); // number of ghost cells in all fields
-        pp_query("geometry.x_len", value.x_len); // Domain x length
-        pp_query("geometry.y_len", value.y_len); // Domain y length
+        pp_query_default("pf.eps", value.pf.eps, 0.0); // Burn width thickness
```

Comment:
@mongole584 A default value of 0.0 for this will cause the model to blow up. Please determine: what was the default value in the original code?